tests/qos/test_buffer.py

```diff
@@ -72,6 +74,19 @@ def detect_buffer_model(duthost):
     BUFFER_MODEL_DYNAMIC = (buffer_model == 'dynamic')
 
 
+def detect_lossy_only_pool(duthost):
```

Comment:
is it possible to extract these 2 functions? I believe we will also need these functions in test_qos_sai and other tests.